nixos/lib/testing-python.nix

```diff
@@ -52,12 +52,13 @@ rec {
     , skipLint ? false
     , passthru ? {}
     , interactive ? false
+    , withAdditionalPythonPackages ? (_ :[])
```

Comment:
I plan to simplify such wiring with the module system. Let's get your change in first.